javascript/README.md

```diff
@@ -402,3 +402,48 @@ Prefer the short, more succinct version.
 Array.prototype.forEach.call(arguments, function(arg) { console.log(arg); });
 ```
 
+##Prefer `self` to `bind(this)`
```

Comment:
I was just looking at some code I wrote. I wanted to pass in a prototype function into another function to be called at a later time. I think this is reasonable time to use `bind(this)`. Here is the line: https://github.com/mobify/astro-scaffold/blob/hyb-578-update-scaffold-header-bar-tabs/app/scaffold-controllers/navigationController.js#L38